media/js/base/global.js

```diff
@@ -114,10 +114,11 @@ function isFirefoxUpToDate(latest) {
     return (latestFirefoxVersion <= fx_version);
 }
 
-// Used on the plugincheck page to also support SeaMonkey
-function isSeaMonkey(userAgent) {
+// Used on the plugincheck page to also support all browsers based on Gecko.
+// Not including Camino as it has reached EOL [http://caminobrowser.org/]
+function isLikeFirefox(userAgent) {
     var ua = userAgent || navigator.userAgent;
-    return /SeaMonkey/i.test(ua);
+    return (/Iceweasel/i).test(ua) || (/SeaMonkey/i).test(ua);
```

Comment:
hmmmm, seems the last two version of Camino switched to having the string 'Firefox compatible) in the user agent string and not 'like Firefox' anymore:

http://useragentstring.com/Camino8.723_id_19875.php